docs/docs/contribute/docs/03.api-docs.md

```diff
@@ -0,0 +1,18 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# 3. Contributing to the API docs
+
+*// TODO //*
```

Comment:
Let's keep this page but remove the *TODO* sections.

Reply:
//TODO// comments removed